spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/SparkUtils.java

```diff
@@ -0,0 +1,210 @@
+   *
+   * @param inputSchema       schema of the received record.
+   * @param featuresToInclude features to be used for training/prediction.
+   * @param featuresToExclude features to be excluded when training/predicting.
+   * @param labelField        field containing the prediction values.
+   */
+  public static void validateConfigParameters(Schema inputSchema, String featuresToInclude,
+                                              String featuresToExclude, String labelField) {
+    if (!Strings.isNullOrEmpty(featuresToExclude) && !Strings.isNullOrEmpty(featuresToInclude)) {
+      throw new IllegalArgumentException("Cannot specify values for both featuresToInclude and featuresToExclude. " +
+                                           "Please specify fields for one");
+    }
+    List<String> fields = getFeatureList(inputSchema, featuresToInclude, featuresToExclude, labelField);
+    for (String field : fields) {
+      Schema.Field inputField = inputSchema.getField(field);
+      Schema schema = inputField.getSchema();
+      Schema.Type features = schema.isNullableSimple() ? schema.getNonNullable().getType() : schema.getType();
+      if (features.equals(Schema.Type.BYTES) || features.equals(Schema.Type.STRING) ||
+        features.equals(Schema.Type.BOOLEAN)) {
+        throw new IllegalArgumentException(String.format("Field to classify must be continuous : int, double, " +
+                                                           "float, long but was %s.", features));
+      }
+      Preconditions.checkArgument(!features.equals(Schema.Type.NULL), "Field to classify must not be of type null");
+    }
+
+  }
+
+  /**
+   * Get the feature list of the features that have to be used for training/prediction depending on the
+   * featuresToInclude or featuresToInclude list.
+   *
+   * @param inputSchema       schema of the received record.
+   * @param featuresToInclude features to be used for training/prediction.
+   * @param featuresToExclude features to be excluded when training/predicting.
+   * @param labelField        field containing the prediction values.
+   * @return feature list to be used for training/prediction.
+   */
+  public static List<String> getFeatureList(Schema inputSchema, String featuresToInclude, String featuresToExclude,
+                                            String labelField) {
+    List<String> fields = new LinkedList<>();
+    List<Schema.Field> inputSchemaFields = inputSchema.getFields();
+    List<String> excludeFeatures = new ArrayList<>();
+
+    if (!Strings.isNullOrEmpty(featuresToInclude)) {
+      fields = Lists.newArrayList(Splitter.on(',').trimResults().split(featuresToInclude));
+      for (String field : fields) {
+        Schema.Field inputField = inputSchema.getField(field);
+        if (inputField == null) {
+          throw new IllegalArgumentException(String.format("Field %s does not exists in the input schema", field));
+        }
+      }
+    } else {
```

Comment:
The else block is used if featuresToExclude is specified. It will also be valid in case both featuresToInclude and featuresToExclude are null.
Please refer design discussion- https://wiki.cask.co/display/CE/Decision+Tree+Regression